buildout_1.rst

```diff
@@ -16,9 +16,9 @@ Topics covered:
 
 .. only:: not presentation
 
-    `Buildout <https://pypi.python.org/pypi/zc.buildout>`_ composes your application for you, according to your rules.
+    `Buildout <https://pypi.python.org/pypi/zc.buildout>`_ configures your application for you, according to your rules.
 
-    To compose your application you must define the eggs you need, which version, what configuration files Buildout has to generate for you, what to download and compile, and so on.
+    To configure your application you must define the eggs you need, which version, what configuration files Buildout has to generate for you, what to download and compile, and so on.
     Buildout downloads the eggs you requested and resolves all dependencies. You might need five different eggs, but in the end, Buildout has to install 300 eggs, all with the correct version in order to resolve all the dependencies.
```

Comment:
Clearly we agree on the concept, just not on the word.  I'm fine reverting my change, I just won't use `compose` verbally because it does not sound right to me.  Configuration is not scope-limited to a single application;  the work we do when we set up a server stack, or even a server farm, is system configuration, not composition.